apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerDataAdapter.cs

```diff
@@ -279,42 +278,6 @@ private SpannerCommand GetBuiltSelectCommand()
         /// <inheritdoc />
         protected override void OnRowUpdating(RowUpdatingEventArgs rowUpdatingEventArgs)
         {
-            if (rowUpdatingEventArgs.Command == null && AutoCreateCommands)
```

Comment:
I don't really understand what this would be doing. Could you edit the PR description to clarify it a bit, just for the record?